packages/agoric-cli/src/commands/gov.js

```diff
@@ -126,8 +150,14 @@ export const makeEconomicCommiteeCommand = (_logger, io = {}) => {
     show(blockInfo);
   };
 
-  ec.command('committee')
-    .description('accept invitation to join the economic committee')
+  cmd
+    .command('committee')
+    .description('accept invitation to join a committee')
+    .requiredOption(
```

Comment:
requiredOptions can have a default value? _bizarre_.

Reply:
I did it that way in case someone adds the option and as an invalid value.